src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir; /* Имя шрифта */
+  src: url(fonts/Avenir-Book.ttf); /* Путь к файлу со шрифтом */
+}
+
+/* background: url('/img.png') no-repeat right center; */
+
+* {
+  font-family: Avenir, sans-serif;
```

Comment:
Better add fonts for body

Reply:
placeholder does not inherit fonts so you should add font straight to placeholder